### ISSUE TITLE
CDN API results

### ISSUE THREAD

**Issue by aaronhans** [cagov/covid19#181]
Would like to discuss options with Chad and Carter before defining solution.

Opportunity: We are making parts of the covid site now which are good candidates for dynamic client side rendering because:
- they display data managed elsewhere
- are not the most important thing on the page

The find your telehealth provider is a good candidate for this.

The prototype uses a json file inside 11ty, a better version would retrieve that data from our API which updated it regularly from the google sheet or master data source.

But the covid site gets massive traffic and we don't currently have a CDN in front of our API endpoints. Should we just put a CDN in front of them or is this an opportunity for the API to write to blob storage with CDN in front of it that we could also use for static asset publishing?

**Comment by aaronhans** [comment]
Thanks for looking into this @ChadABratton

**Comment by aaronhans** [comment]
I am canceling this. we went a totally different direction here because the data behind telehealth was updated far less frequently than we anticipated.

Current approach: 

- We have a script in our API repo which pulls that latest dataset out of the google spreadsheet.
- We run this script whenever the SME lets us know the data is updated
- Our script creates a json file which we publish as part of our API
